src/mz390.java

```diff
@@ -8492,7 +8493,7 @@ private void init_gbl_sys(){
 		gbl_setc[gbl_sysm_sev_index] = "000"; // RPI 898
 		add_gbl_sys("&SYSOPT_DBCS",var_setb_type); // set false indicating no DBCS support
 		add_gbl_sys("&SYSOPT_OPTABLE",var_setc_type); // "z390" opcode table name
-		gbl_setc[tot_gbl_setc-1] = "z390";
+        gbl_setc[tot_gbl_setc-1] = tz390.opt_optable; // #527
```

Comment:
The next line, while not part of your changes, is messed up. It starts with a semicolon. If you feel like it, clean it up by removing the leading semicolon and align the remaining text with the surrounding lines. Just a suggestion.